gridstatus/ercot_60d_utils.py

```diff
@@ -353,9 +353,18 @@ def process_as_offer_curves(df):
             )
 
             if len(subset) > 1:
-                raise ValueError(
-                    f"More than one row found for {column_list} for {resource_name}",
-                )
+                # We've identified an issue with this specific resource name where
+                # there are sometimes multiple offers for the same service at the same
+                # interval. In theory this should never happen. The QUANTITY MW are
+                # only different by 0.1, so we just take the row with the lowest
+                # quantity. This is a temporary fix until we can figure out why this
+                # is happening.
+                if resource_name == "CANYONRO_LD1":
+                    subset = subset.sort_values("QUANTITY MW1").iloc[0]
```

Comment:
Choosing the row with the lowest `QUANTITY MW1` is arbitrary. When cross-referencing with another dataset (SCED Load resource 60 day) there wasn't a reliable way to determine which row was "correct". The values in `QUANTITY MW1` are only different by 0.1.